README.md

```diff
@@ -1,14 +1,6 @@
 ## Project title
 
-Music Library - exercise for SQL, databases and **CRUD** operations.
-
-## Motivation
-
-- Learn how to design test for each operation.
-- Learn how to route user request to handler.
-- Learn how to handle error of user request.
-
-## What will learn from this exercise
+Music Library - SQL, Databases, Express.js, Routing.
```

Comment:
There are a few issues with this README. The first issue is that from the commit history it contains David Cheung's name in a few places which have later been replaced with your own and when comparing your README with his it is almost identical. While we encourage collaborating with other students we don't want copy and pasting and this is your learning experience we want you to get the best out of it. If you need inspiration for a README it would be best to read lots of different ones to help give you some ideas of content and layout.
This README is actually missing some very key information and has information in it that really doesn't need to be there. Things to include are
1) A description of the project, why you are doing it, technologies used, dependencies, dev dependencies
2) How to download and use the app, think of everything they would need to do this (including things like docker containers, pgadmin and more)
3) A list a routes available with the endpoints, the route parameters, what data needs to be included in post and patch request